backend/tests.py

```diff
@@ -123,4 +123,15 @@ def test_get_roll(client, drone_controller):
         assert "status" in data
         assert data["status"] == "success"
         assert "roll" in data
-        assert data["roll"] == 1500
\ No newline at end of file
+        assert data["roll"] == 1500
+
+def test_disarm_drone(client, drone_controller):
```

Comment:
@vivekanandareddy-ponugoti Thanks for adding the test case. The test case has passed. I am merging the pull request. to create a release file